crates/sui/tests/shared_objects_version_tests.rs

```diff
@@ -15,9 +14,7 @@ use sui_types::SUI_FRAMEWORK_ADDRESS;
 use test_utils::authority::{spawn_test_authorities, test_authority_configs};
 use test_utils::messages::move_transaction;
 use test_utils::objects::test_gas_objects;
-use test_utils::transaction::{
-    publish_package, submit_shared_object_transaction, submit_single_owner_transaction,
-};
+use test_utils::transaction::{publish_package, submit_single_owner_transaction};
 
 #[sim_test]
 async fn fresh_shared_object_initial_version_matches_current() {
```

Comment:
@mwtian, this test is the happy path for initial shared version -- it's intended to be populated with the version at which the version becomes shared.  With the introduction of lamport timestamps, this is no longer always `1` (even in the absence of owned to shared promotion).

Reply:
Sounds good.